components/ViewPhotos.js

```diff
@@ -20,15 +22,13 @@ export default class ViewPhotos extends Component {
 	render(){
 		let { image } = this.state;
 		return (
-			<View style={style.iewPhotos}>
-			<FormLabel>Screenshot</FormLabel>
-				<Button
-					title="Pick an image from camera roll"
-					onPress={this._pickImage}
-				/>
+			// <FormLabel>Screenshot</FormLabel>
+			<View style={style.viewPhoto}>
+			<View><UploadPhotoButton label='Add a photo' color='rgb(0,175,115)' onPress={this._pickImage}/></View>
+
 				{image &&
 					<Image source={{ uri: image }} />}
-			</View>
+			// </View>
```

Comment:
Why is this View tag commented out? Where's the closing View tag for the one on line 26?

Reply:
It wasn't commented out, because of JSX comment rules, but why was it like that?